s3-uploader/runtimes/cpp11/Dockerfile

```diff
@@ -1,26 +1,36 @@
-FROM public.ecr.aws/lambda/provided:al2 as builder
+ARG ARCH
+FROM public.ecr.aws/lambda/provided:al2-$ARCH AS builder
 ENV APP_BASE_DIR=/tmp/app
 WORKDIR $APP_BASE_DIR
-RUN yum install cmake3 make g++ git zip libcurl-devel zlib-devel openssl-devel -y
+RUN yum install cmake3 git zip autoconf automake -y
 RUN yum groupinstall "Development Tools" -y
 
 COPY . $APP_BASE_DIR
 WORKDIR $APP_BASE_DIR
 SHELL ["/bin/bash", "-c"]
+# Build static libcurl from source
+RUN curl -O https://curl.se/download/curl-7.88.1.tar.gz
+RUN tar -xvf curl-7.88.1.tar.gz
+WORKDIR "${APP_BASE_DIR}/curl-7.88.1"
+RUN autoreconf -fi
+RUN ./configure --disable-shared --disable-pthreads --disable-threaded-resolver \
+--with-pic --without-ssl --without-zlib --prefix=${APP_BASE_DIR}/lambda-install
```

Comment:
could you please update the prefix to `--prefix=/usr` so both libs and includes will be in the path

Reply:
I removed it. `/usr` is the default. The install prefix is not needed in any of the Lambda creation either, but I didn't change those.

Reply:
Thanks!